specification/service-search-api.yaml

```diff
@@ -18,4 +157,483 @@ servers:
     description: Integration test environment.
   - url: 'https://api.service.nhs.uk/service-search-api'
     description: Production environment.
-paths: {}
+paths:
+
+  "":
+  
+    # --------------------------------------------------------------------------------
+    # GET /
+    # --------------------------------------------------------------------------------
+  
+    get:
+      operationId: root-GET
+      description: Information about services provided by NHS organisations listed in the service search on the NHS website.
+      parameters:
+        - name: api-version
+          description: The API version number can be supplied as a query parameter.
+          in: query
+          schema:
+            type: number
+            default: 1
+            enum: [1]
+        - name: search
+          in: query
+          description: The text to search for. All searchable fields are searched by default unless searchFields is specified. When searching searchable fields, the search text itself is tokenized, so multiple terms can be separated by white space (for example search=hello world). To match any term, use * (this can be useful for boolean filter queries). Omitting this parameter has the same effect as setting it to *.
+          schema:
+            type: string
+        - name: searchMode
+          in: query
+          description: Defaults to any. Specifies whether any or all of the search terms must be matched in order to count the document as a match.
+          schema:
+            type: string
+            enum: 
+            - 'any'
+            - 'all'
+            default: 'any'
+        - name: searchFields
+          in: query
+          description: The list of comma-separated field names to search for the specified text.
+          schema:
+            type: string
+        - name: queryType
+          description: Defaults to simple. When set to simple, search text is interpreted using a simple query language that allows for symbols such as +, * and "". When the query type is set to full, search text is interpreted using the Lucene query language which allows field-specific and weighted searches.
+          in: query
+          schema:
+            type: string
+            enum:
+            - 'simple'
+            - 'full'
+            default: 'simple'
+        - name: $skip
```

Comment:
Is this default 0?

Reply:
0?  I don't follow.

Reply:
The comment is for the block contained in $skip (line 207)